topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -2082,7 +2082,7 @@ For this, we will use NGINX (pronounced "engine X" /ˌɛndʒɪnˈɛks/ EN-jin-EK
 >
 >    > <details-title>Running this tutorial <i>without</i> SSL</details-title>
 >    >
->    > In your `galaxy.j2` in the above step, you should change the `listen` parameter:
+>    > Ignore the https redirection by omitting `templates/nginx/redirect-ssl.j2`. In your `galaxy.j2` in the above step, you should change the `listen` parameter:
```

Comment:
```suggestion
>    > In your `galaxy.j2` in the above step, you should change the `listen` parameter:
```

This is covered in the other "Running this tutorial without SSL details box.

Reply:
I saw the second box, but it does not fix my problem. If I follow both boxes, I get a 502 error.

Reply:
That's quite odd! 502 indicates an error reaching the Galaxy backend, if you're seeing that it usually isn't related to nginx, nor ssl configuration, but instead because galaxy isn't started properly (maybe the conda issue?)

The first box includes precisely this change, removing the redirect from deployment. If you followed the tutorial as written, then you saw the first box which included this change:

![image](https://github.com/galaxyproject/training-material/assets/458683/584dac43-d35b-47a5-a7c4-f4358f8a3265)

that would prevent the redirect file from ever being sent to the server where galaxy is being set up. The template will still exist in your ansible, but, it isn't included in the configuration.